ci/dra_common.sh

```diff
@@ -0,0 +1,33 @@
+function info {
+    echo "INFO: $1"
+}
+
+function error {
+    echo "ERROR: $1"
+}
+
+function save_docker_tarballs {
+    ver=$1
+    arch=$2
+    for image in logstash logstash-oss logstash-ubi8; do
+        # docker save docker.elastic.co/logstash/logstash:${STACK_VERSION} | gzip -c > build/logstash-${STACK_VERSION}-docker-image-x86_64.tar.gz
+        # docker save docker.elastic.co/logstash/logstash-oss:${STACK_VERSION} | gzip -c > build/logstash-oss-${STACK_VERSION}-docker-image-x86_64.tar.gz
+        # docker save docker.elastic.co/logstash/logstash-ubi8:${STACK_VERSION} | gzip -c > build/logstash-ubi8-${STACK_VERSION}-docker-image-x86_64.tar.gz
+        docker save docker.elastic.co/logstash/$image:$ver | gzip -c > build/$image-$ver-docker-image-$arch.tar.gz
+    done
```

Comment:
To make this easier to understand, I prefer to:
 - quote args that contain variables (prevents spaces in var values from splitting into multiple args during parameter expansion which can be very tricky to debug),
 - use the `${varname}` syntax to clearly delineate where the varname ends (my brain tracks it a _lot_ better than remembering exactly which characters are allowed in varnames)
 - use the `${varname:?error message if unset}` syntax when capturing positional or required args to make failure to provide it easier to detect/debug (prevents a later failure when using an empty variable)

```suggestion
    ver="${1:?version required}"
    arch="${2:?arch required}"
    for image in logstash logstash-oss logstash-ubi8; do
        # docker save docker.elastic.co/logstash/logstash:${STACK_VERSION} | gzip -c > build/logstash-${STACK_VERSION}-docker-image-x86_64.tar.gz
        # docker save docker.elastic.co/logstash/logstash-oss:${STACK_VERSION} | gzip -c > build/logstash-oss-${STACK_VERSION}-docker-image-x86_64.tar.gz
        # docker save docker.elastic.co/logstash/logstash-ubi8:${STACK_VERSION} | gzip -c > build/logstash-ubi8-${STACK_VERSION}-docker-image-x86_64.tar.gz
        docker save "docker.elastic.co/logstash/${image}:${ver}" | gzip -c > "build/${image}-${ver}-docker-image-${arch}.tar.gz"
    done
```